traits/tests/test_listeners.py

```diff
@@ -37,7 +38,7 @@ def captured_stderr():
     Return a context manager that directs all stderr output to a string.
 
     """
-    new_stderr = cStringIO.StringIO()
+    new_stderr = sm.cStringIO()
```

Comment:
changing to just `six.StringIO` for consistency with similar uses elsewhere in the codebase.